autotest/ogr/ogr_lvbag.py

```diff
@@ -0,0 +1,165 @@
+       feat.GetFieldAsString(9) != '2009-01000':
+        feat.DumpReadable()
+        pytest.fail()
+
+    feat = lyr.GetNextFeature()
+    feat = lyr.GetNextFeature()
+    feat = lyr.GetNextFeature()
+    assert feat is None
+
+def test_ogr_lvbag_2():
+
+    ds = ogr.Open('data/lvbag/num.xml')
+    assert ds is not None, 'cannot open dataset'
+    assert ds.GetLayerCount() == 1, 'bad layer count'
+    assert ds.GetLayer(1) is None
+
+    lyr = ds.GetLayer(0)
+    assert lyr.GetName() == 'Nummeraanduiding', 'bad layer name'
+
+    assert lyr.GetGeomType() == ogr.wkbUnknown, 'bad layer geometry type'
+    assert lyr.GetSpatialRef() is None, 'bad spatial ref'
+    assert lyr.GetFeatureCount() == 3
+    assert lyr.TestCapability(ogr.OLCStringsAsUTF8) == 1
+
+    assert lyr.GetLayerDefn().GetFieldCount() == 13
+
+    feat = lyr.GetNextFeature()
+    if feat.GetField('namespace') != 'NL.IMBAG.Nummeraanduiding' or \
+       feat.GetField('lokaalID') != '0106200000002798' or \
+       feat.GetField('huisnummer') != '23' or \
+       feat.GetField('postcode') != '9403KB' or \
+       feat.GetField('typeAdresseerbaarObject') != 'Verblijfsobject' or \
+       feat.GetField('status') != 'Naamgeving uitgegeven' or \
+       feat.GetField('geconstateerd') != 'N' or \
+       feat.GetFieldAsString('documentdatum') != '2009-09-14' or \
+       feat.GetFieldAsString('documentnummer') != '2009-BB01570' or \
+       feat.GetField('voorkomenidentificatie') != '1' or \
+       feat.GetField('beginGeldigheid') != '2009-09-24' or \
+       feat.GetField('tijdstipRegistratie') != '2009-11-06T12:21:37.000' or \
+       feat.GetField('tijdstipRegistratieLV') != '2009-11-06T12:38:46.603':
+        feat.DumpReadable()
+        pytest.fail()
+
+    feat = lyr.GetNextFeature()
+    feat = lyr.GetNextFeature()
+    feat = lyr.GetNextFeature()
+    assert feat is None
+
+def test_ogr_lvbag_3():
+
+    ds = ogr.Open('data/lvbag/opr.xml')
+    assert ds is not None, 'cannot open dataset'
+    assert ds.GetLayerCount() == 1, 'bad layer count'
+
+    lyr = ds.GetLayer(0)
+    assert lyr.GetName() == 'OpenbareRuimte', 'bad layer name'
+
+    assert lyr.GetGeomType() == ogr.wkbUnknown, 'bad layer geometry type'
+    assert lyr.GetSpatialRef() is None, 'bad spatial ref'
+    assert lyr.GetFeatureCount() == 3
+    assert lyr.GetLayerDefn().GetFieldCount() == 12
+
+def test_ogr_lvbag_4():
+
+    ds = ogr.Open('data/lvbag/pnd.xml')
+    assert ds is not None, 'cannot open dataset'
+    assert ds.GetLayerCount() == 1, 'bad layer count'
+
+    lyr = ds.GetLayer(0)
+    assert lyr.GetName() == 'Pand', 'bad layer name'
+
+    assert lyr.GetGeomType() == ogr.wkbPolygon25D, 'bad layer geometry type'
+    assert '28992' in lyr.GetSpatialRef().ExportToWkt()
+    assert lyr.GetFeatureCount() == 6
+    assert lyr.GetLayerDefn().GetFieldCount() == 14
+
+###############################################################################
+# Run test_ogrsf
+
+
+def test_ogr_lvbag_4():
```

Comment:
```suggestion
def test_ogr_lvbag_test_ogrsf():
```
(to avoid the accidental duplication of test names)